src/components/icon/icons/add_part.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M3 0C1.34315 0 0 1.34315 0 3V8C0 9.65685 1.34315 11 3 11H8C9.65685 11 11 9.65685 11 8V3C11 1.34315 9.65685 0 8 0H3Z" fill="#currentColor"/>
```

Comment:
`fill` attributes should have `currentColor` (without `#`)

This applies for all files.